pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/submission/Issue.kt

```diff
@@ -0,0 +1,15 @@
+package gov.cdc.ocio.processingstatusapi.models.submission
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+
+/**
+ * Issues within a report.
+ *
+ * @property items Items?
+ */
+@GraphQLDescription("Contains Report issues.")
+data class Issue(
+
+    @GraphQLDescription("Items")
+    var items : Items? = null
```

Comment:
The `Issue` class should only contain `level` and `message`.  The `Items` and `Properties` classes aren't needed.